app/views/steps/client/nino_exit/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="flash-card">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <header>
+        <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
+      </header>
+
+      <p>You cannot use this service if your client:</p>
```

Comment:
On thing I usually do with heavy-copy views (and in particular these kind of show views without any particular logic or complexity) is to use translated views (as in `show.en.html.erb`) and use hardcoded copy (no locales, apart from perhaps just the `page_title`).

It makes these views easier to create, to read and to modify. If we ever have another language, we make a copy of the view, with a new locale (i.e. show.cy.html.erb) and just translate the copy on it.

Just a suggestion! 😄